lib/gateway.js

```diff
@@ -0,0 +1,39 @@
+export function extractExplainGateway(context, { profiler, resolverCalls }) {
+  if (!context.collectors?.extensions) {
+    console.warn('Collectors are empty')
```

Comment:
I haven't dug into the code yet but from a quick glance it seems we should be using a logger instance here rather than `console`.

Reply:
do you think using the `context.app.log.warn('Mercurius explain is not enabled on any service')` is better?

Reply:
yes, that seems to be a better way.